log.c

```diff
@@ -184,10 +181,14 @@ static void __vlog(char *module, int level, const char *fmt, va_list args)
 			int err_fd = -1;
 
 
-			snprintf(err_file, PATH_MAX, "%s/%s", log_dir, ERROR_DIR);
+			if (PATH_MAX <= snprintf(err_file, sizeof(err_file), "%s/%s",
```

Comment:
isn't this the same we do in sprintf_path?

Reply:
Originally, this was an oversight on my part. But now because SNPRINTF_WTRUNC uses pv_log, I kept it this way: Better not use SNPRINTF_WTRUNC in log.c, because it could lead to infintite recursion.